_source/_posts/2021-11-19-oie-hackathon-winner.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: blog_post
+title: "Getting to Know GooseClip: Interview with an Okta Hackathon Winner"
+author: [heather-downing, brian-demers]
+by: advocate
+communities: [go]
+description: "Okta Developer Advocates Brian Demers and Heather Downing in conversation with Matthew McKenzie, developer of GooseClip, a peer-to-peer app which took first prize in the Okta's Identity Engine Hackathon."
+tags: [go, okta, announcement, hackathon]
+tweets:
+- ""
+- ""
+- ""
+image: blog/oie-hackathon-winner/social.jpg
+type: awareness 
+---
+
+## Okta Early Access Hackathon revisited
+
+In August, Okta's developer relations team invited intrepid developers to participate in [the virtual Okta Identity Engine Early Access Hackathon](https://developer.okta.com/blog/2021/08/11/okta-identity-engine-early-access-hackathon), and challenged them to build a working app with our limited availability new customer identity and access management (CIAM) platform. A month or so after, judges convened, [winners were announced](https://developer.okta.com/blog/2021/09/28/okta-hackathon-2021-winners), and thousands of dollars in prizes were distributed. Kudos to hundreds of participants, and props to all involved. 
```

Comment:
Run `npm run clean-post` to convert these absolute to relative URLs.

Reply:
good catch! I didn't realize that was in `clean-post` but not in the pre-commit linting

To attempt to catch this in the future: #969